server/mdm/microsoft/admx/admx_test.go

```diff
@@ -13,6 +13,37 @@ func TestIsADMX(t *testing.T) {
 	assert.False(t, IsADMX(`<![CDATA[bozo]]>`))
 	assert.False(t, IsADMX(`<![CDATA[<bozo/>]]>`))
 	assert.False(t, IsADMX(`<![CDATA[<bozo]]>`))
+
+	// This is a WLAN XML profile
+	assert.False(t, IsADMX(
+		`&lt;?xml version=&quot;1.0&quot;?&gt;
```

Comment:
Just curious (I don't know what is ADMX?), why does the XML has to be escaped here? (i.e. why don't we send `<?xml...`?)

Reply:
Oh I think it's because that's how we receive it from the host for verification, right?